client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEdit.tsx

```diff
@@ -53,6 +55,11 @@ export const StocktakeLineEdit: FC<StocktakeLineEditProps> = ({
   const theme = useAppTheme();
   const isMediumScreen = useMediaQuery(theme.breakpoints.down(Breakpoints.lg));
   const [currentItem, setCurrentItem] = useState(item);
+
+  const { data: preferences } = usePreference(
+    PreferenceKey.AllowTrackingOfStockByDonor
+  );
```

Comment:
Moved out of the columns hook - pref hadn't loaded in time and the column wasn't rendering - suppose other option would be to put the prefs as a useColumns dependency, but I think putting it here means we query less often so probably better anyway?

Reply:
I would even move it to the parent component 👀 even lesser calls~. All the prefs should be loaded at parent level 👀